Readme_ES.md

```diff
@@ -0,0 +1,307 @@
+# Paquete GT para moderadores de New Horizons
+
+## Paquete para moderadores verificado a través de un ajuste a los moderadores (LOS MODERADORES OBTIENEN AUTORIZACIÓN)
```

Comment:
Another one: "Package for moderators verified through a setting to the moderators (THE MODERATORS OBTAIN AUTHORIZATION)".